deploy/sdk/src/dynamo/sdk/cli/build.py

```diff
@@ -128,6 +128,7 @@ def from_service(cls, service: ServiceInterface[T]) -> ServiceInfo:
         for ep_name, endpoint in service.get_dynamo_endpoints().items():
             if DynamoTransport.HTTP in endpoint.transports:
                 api_endpoints.append(f"/{ep_name}")
+        http_exposed = service.config.http_exposed or len(api_endpoints) > 0
```

Comment:
What if we only expose endpoints if the user explicitly specifies `http_exposed`? If the user has `@api` or `@endpoint` API endpoints but doesn't set `http_exposed`, HTTP doesn't get exposed. This would also need to be reflected in service.py.